ui/DesignSystem/Component/AdditionalActionsDropdown.svelte

```diff
@@ -121,7 +121,7 @@
     <div out:fade={{ duration: 100 }} class="modal" hidden={!expanded}>
       {#if headerTitle}
         <div class="header">
-          <Urn urn={headerTitle} />
+          <Urn urn={headerTitle} notificationTxt="Copied to your clipboard" />
```

Comment:
Could we call the prop `notificationText`?

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/758/commits/0b9867696a1ef5ce4d154ef1d30462fa65af6006